src/autoloads/appearance.gd

```diff
@@ -1,10 +1,11 @@
 extends Node
 
-signal applyConfig # Apply config for menu character
+signal configUpdated # Apply config for menu character
```

Comment:
"config" is not the good word here. I definitely wouldn't associate from the word "config" to the appearance.
also, signals are function calls, so verb + noun makes sense for me, but that could be debated.

changeOwnOutfit or changeOwnAppearance would be better choices IMHO even if longer words.

Reply:
ChangeOwnOutfit wont work as outfit is separate to colours, i jad config meaning outfit + colours, but i guess we can change to updateAppearance.